### PR TITLE
Update to urfave cli v2

### PR DESCRIPTION
**Pull request by refs**
This also fixes the way we parsed the `web-config-apps` flag.

Old behaviour:

`ocis-phoenix server --web-config-apps "files,draw-io,pdf-viewer"`

New behaviour:

`ocis-phoenix server --web-config-apps files --web-config-apps pdf-viewer`

### CODE REVIEW

**Comment by tboerger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pkg/command/server.go  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis-phoenix/pullRequest?prid=4889471&bid=16194974)

File: pkg/command/server.go [hunk 1]
```diff
@@ -35,15 +34,13 @@ func Server(cfg *config.Config) cli.Command {
 				cfg.HTTP.Root = strings.TrimSuffix(cfg.HTTP.Root, "/")
 			}
 
+			cfg.Phoenix.Config.Apps = c.StringSlice("web-config-apps")
```
Comment (tboerger) [inline pkg/command/server.go]: I personally put this assignment normally into `Before`, than at least it feels more consistent until StringSliceFlag also gets the Destination keyword.

File: pkg/flagset/flagset.go [hunk 2]
```diff
@@ -56,159 +58,158 @@ func ServerWithConfig(cfg *config.Config) []cli.Flag {
...
+			EnvVars:     []string{"PHOENIX_WEB_CONFIG_VERSION"},
 			Destination: &cfg.Phoenix.Config.Version,
 		},
-		&cli.StringFlag{
-			Name:   "web-config-apps",
-			Value:  "files,draw-io,pdf-viewer,markdown-editor,media-viewer",
-			Usage:  `String with comma separated values. --web-config-apps "pdf-viewer, files, draw-io"`,
-			EnvVar: "PHOENIX_WEB_CONFIG_APPS",
+		&cli.StringSliceFlag{
+			Name:    "web-config-apps",
```
Comment (tboerger) [inline pkg/flagset/flagset.go]: Maybe name it in singular? That would show it a little bit more that you got to provide this flag per app.

File: pkg/command/server.go (hunk 1 above)
Comment (refs) [reply pkg/command/server.go]: it is in the before callback

File: pkg/flagset/flagset.go (hunk 2 above)
Comment (refs) [reply pkg/flagset/flagset.go]: agree

File: pkg/command/server.go (hunk 1 above)
Comment (tboerger) [reply pkg/command/server.go]: Oh.... GitHub tricked me, sorry :D

Comment (refs) [reply pkg/command/server.go]: hahaha

**Review by tboerger — APPROVED** [review]

**Comment by tboerger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pkg/command/server.go  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis-phoenix/pullRequest?prid=4889471&bid=16194974)

**Pull request merged**